README.md

```diff
@@ -116,6 +116,22 @@ Hint 4
 --- /hints ---
 ```
 
+### Quiz
+
+Quizzes can be added with choices for the user to select (currently only 1 mutually exclusive choice per quiz):
+``` markdown
+--- quiz ---
+---
+question: Here is a heading for a quiz with three possible answers. How do you feel?
+---
+
+- () Great
+- () Okay
+- () Terrible
```

Comment:
This should probably match the examples/quiz/quiz.md and have spaces..

Sorry..!